Externals.cfg

```diff
@@ -50,40 +50,40 @@ required = True
 #-----------------------------
 
 [blom]
-tag = 81cd404
 protocol = git
+tag = 2bb8e7c
```

Comment:
It looks like this is a hash. I would use the full hash because the short version can become inadequate over time (i.e., git would provide more digits when a conflict shows up).